packages/odyssey-react-mui/src/Banner.tsx

```diff
@@ -0,0 +1,64 @@
+/*!
+ * Copyright (c) 2022-present, Okta, Inc. and/or its affiliates. All rights reserved.
+ * The Okta software accompanied by this notice is provided pursuant to the Apache License, Version 2.0 (the "License.")
+ *
+ * You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0.
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *
+ * See the License for the specific language governing permissions and limitations under the License.
+ */
+
+import { AlertColor } from "@mui/material";
+import { Alert, Link } from "./";
+
+export interface BannerProps {
+  /**
+   * Determine the color and icon of the alert
+   */
+  severity?: AlertColor;
+  /**
+   * Sets the ARIA role of the alert
+   * ("status" for something that dynamically updates, null for something
+   * unchanging)
+   */
+  role?: "status" | undefined;
+  /**
+   * The text content of the alert
+   */
+  children: string;
```

Comment:
`children` should technically never be a string. People will assume you can pass React components in here since just about every `children` is type `ReactNode`. That's how HTML works in general.

I'd recommend using a `text` or `description` prop instead for string-only children just to avoid breaking convention.

Please give me your thoughts. We can meet on this one.

Reply:
That makes sense to me, let's switch to `text`.